src/components/Filter/PunchListFilter/usePunchListFilterFacade.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { useEffect } from 'react';
+import { useState } from 'react';
+import { PunchPreview } from '../../../services/apiTypes';
+import useFilterFacade, { Filter } from '../useFilter';
+import { filterPunchPreviews } from './filterPunchPreviews';
+
+// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
```

Comment:
hmmm.. Ikke bra. ville lagt til return type

Reply:
Her har vist Erlend satt en standard. Vi lar det går for nå så kan vi diskutere dette på et senere tidspunkt :)